migrations/2022_05_09_233348_create_comments_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('comments', function (Blueprint $table) {
+            $table->id();
+            $table->string('content');
+            $table->integer('player_id');
+            $table->integer('user_id');
```

Comment:
set relations for those
```
$table->index("user_id");
            $table->foreign("user_id")
                ->references('id')->on('users')
                ->onDelete('cascade');
```

The above sample first creates the index on the field and then adds the foreign key relation. If also any other field requires index eg for faster search make sure you add this as well